openapi/openapi.yaml

```diff
@@ -2167,23 +2167,27 @@ components:
     QebHwtThamosAdviseInput:
       type: object
       required:
-        - check_run_id
-        - repo_url
-        - commit_sha
-        - installation_id
+        - github_event_type
+        - github_check_run_id
+        - origin
+        - revision
+        - github_installation_id
       properties:
-        check_run_id:
+        github_event_type:
+          type: integer
```

Comment:
https://github.com/thoth-station/common/pull/648 `github_event_type` is a string